tidb/metadata.csv

```diff
@@ -1,43 +1,13 @@
-tidb_cluster.tidb_pd_client_request_handle_requests_duration_seconds.count,count,,,,The total number of requests from PD client,1,tidb,
-tidb_cluster.tidb_pd_client_request_handle_requests_duration_seconds.sum,count,,,,The sum of request duration from PD client,1,tidb,
-tidb_cluster.tidb_pd_client_cmd_handle_cmds_duration_seconds.count,count,,,,The total number of commands from PD client,1,tidb,
-tidb_cluster.tidb_pd_client_cmd_handle_cmds_duration_seconds.sum,count,,,,The sum of command duration from PD client,1,tidb,
-tidb_cluster.tidb_domain_load_schema_duration_seconds.count,count,,,,The total number of domain loaded schemas,1,tidb,
-tidb_cluster.tidb_domain_load_schema_duration_seconds.sum,count,,,,The sum of domain schema loading duration,1,tidb,
-tidb_cluster.tidb_go_memstats_heap_inuse_bytes,count,,,,Go runtime inuse memory,1,tidb,
-tidb_cluster.tidb_process_resident_memory_bytes,count,,,,Go runtime process resident memory,1,tidb,
-tidb_cluster.pd_tso_events,gauge,,,,The number of tso events,1,tidb,
-tidb_cluster.pd_cluster_status,gauge,,,,PD cluster status,1,tidb,
-tidb_cluster.pd_regions_status,gauge,,,,PD region status,1,tidb,
-tidb_cluster.pd_hotspot_status,gauge,,,,PD hotspot status,1,tidb,
-tidb_cluster.pd_scheduler_region_heartbeat,gauge,,,,PD region scheduler heartbeat,1,tidb,
-tidb_cluster.pd_grpc_server_handling_seconds.sum,count,,,,The total number of grpc request PD handled,1,tidb,
-tidb_cluster.pd_grpc_server_handling_seconds.count,count,,,,The sum of duration for grpc requests PD handled,1,tidb,
-tidb_cluster.pd_scheduler_region_heartbeat_latency_seconds.sum,count,,,,The total number of scheduler region heartbeat,1,tidb,
-tidb_cluster.pd_scheduler_region_heartbeat_latency_seconds.count,count,,,,The sum of duration for scheduler region heartbeat,1,tidb,
-tidb_cluster.tikv_raft_store_region_count,count,,,,Region count of raft stores,1,tidb,
-tidb_cluster.tikv_thread_cpu_seconds_total,count,,,,Sum of cpu seconds,1,tidb,
-tidb_cluster.tikv_engine_size_bytes,count,,,,Sum of engine size,1,tidb,
-tidb_cluster.tikv_channel_full_total,count,,,,Sum of channel full events,1,tidb,
-tidb_cluster.tikv_server_report_failure_msg_total,count,,,,Sum of server report failures,1,tidb,
-tidb_cluster.tikv_scheduler_context_total,count,,,,Sum of scheduler contexts,1,tidb,
-tidb_cluster.tikv_coprocessor_executor_count,count,,,,Sum of coprocessor executors,1,tidb,
-tidb_cluster.tikv_coprocessor_request_duration_seconds.sum,count,,,,Total number of coprocessor requests,1,tidb,
-tidb_cluster.tikv_coprocessor_request_duration_seconds.count,count,,,,Sum of coprocessor request duration,1,tidb,
-tidb_cluster.tidb_session_parse_duration_seconds.sum,count,,,,Total number of session parses,1,tidb,
-tidb_cluster.tidb_session_parse_duration_seconds.count,count,,,,Sum of session parse duration,1,tidb,
+tidb_cluster.tidb_server_handle_query_duration_seconds,count,,,,The distribution of TiDB query durations,1,tidb,
```

Comment:
```suggestion
tidb_cluster.tidb_server_handle_query_duration_seconds,count,,second,,The distribution of TiDB query durations,1,tidb,
```
It's not required, but could you add units for all of the metrics?

Reply:
`tidb_cluster.tidb_server_handle_query_duration_seconds` is actually `distribution` type.

How can I define `distribution` metric type? I got an error 

> "distribution" is an invalid metric_type 

when I tried to set type to distribution.

I also checked the validation source code and got this:
```python
VALID_METRIC_TYPE = {'count', 'gauge', 'rate'}
```

Reply:
Can you revert back to specifying the two metrics you defined previously? These count and sum metrics are sent by the check for the distribution 
```
tidb_cluster.tidb_server_handle_query_duration_seconds.count,count,,,,The total number of handled queries in server,1,tidb,
tidb_cluster.tidb_server_handle_query_duration_seconds.sum,count,,,,The sum of handled query duration in server,1,tidb,
```